eng/tools/sdk-suppressions/src/index.ts

```diff
@@ -0,0 +1,31 @@
+
+import { exit } from "process";
+import { updateSdkSuppressionsLabels, PullRequestContext } from "./updateSdkSuppressionsLabel.js";
+
+function getUsage(): string {
+    return (
```

Comment:
Tools under `eng/tools` must be first-and-foremost designed for a dev to manually run on their dev machine.  Requiring env vars like `GITHUB_PULL_REQUEST_CONTEXT` is not appropriate for dev inner loop.

Instead, you need to take parameters like a "base" and "target" commitish, so devs can manually compare two commits, to reproduce what they see in their PR check.

Example: https://github.com/Azure/azure-rest-api-specs/blob/main/eng/scripts/TypeSpec-Requirement.ps1#L3-L6

Reply:
remove process.env in code and instead of args.